engineering/handlebars.md

```diff
@@ -32,6 +33,25 @@ or helper name.
   value=blah
   options=options
   label="thing"}}
+  
+{{! good }}
+{{#x-thing
+  value=blah
+  options=options
+  label="thing"
+  as |some thing|
+}}
+  {{some (action thing)}}
+{{/x-thing}}
```

Comment:
Agree. I don't like that other indentation as well 👍 
To be honest maybe it was my confusion, I was doing that for multi-line *block* expressions and not for the "not block"(?) ones. Should we really indent 4 spaces in this last case? I don't think I've seen we using that indentation. Maybe we can adjust that here as well?